e2e/local/util.go

```diff
@@ -58,5 +59,8 @@ func kamelWithContext(ctx context.Context, args ...string) *cobra.Command {
 	if runtimeRepo != "" {
 		args = append(args, "--maven-repository", runtimeRepo)
 	}
+	if runtimeVersion != "" {
```

Comment:
Do you mean `if strings.HasSuffix(runtimeVersion, "-SNAPSHOT")`?  Otherwise the snapshot repo should be always setup.

Reply:
Good one. Clearly a mistake. Thanks!